compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFinder.java

```diff
@@ -1711,6 +1729,41 @@ public void visit(BLangReFlagExpression reFlagExpression) {
         lookupNode(reFlagExpression.colon);
     }
 
+    private void lookupResourceAccessPathSegments(BLangInvocation.BLangResourceAccessInvocation resourceInvocation) {
+        if (resourceInvocation.targetResourceFunc == null) {
+            // Return if target-function is not set.
+            // Ex: Ambiguous target-functions
+            return;
```

Comment:
Codecov warning?

Reply:
Added a test for ambiguous resource method